src/components/scene/renderer.js

```diff
@@ -0,0 +1,82 @@
+/* global HTMLElement */
+var register = require('../../core/component').registerComponent;
+var debug = require('../../utils/debug');
+
+var warn = debug('components:renderer:warn');
+
+/**
+ * Determines state of various renderer properties.
+ */
+module.exports.Component = register('renderer', {
+  schema: {
+    antialias: {default: 'auto', oneOf: ['true', 'false', 'auto']},
+    gammaOutput: {default: false},
+    physicallyCorrectLights: {default: false},
+    sortObjects: {default: false}
+  },
+  init: function () {
+    var el = this.el;
+    var data = this.data;
+    var renderer = el.renderer;
+
+    if (!el.isScene) {
+      warn('Fog component can only be applied to <a-scene>');
+    }
+
+    if (el.hasAttribute('antialias')) {
+      warn('Component `antialias` is deprecated. Use `renderer="antialias: true"` instead.');
+    }
+
+    // Default not AA for mobile.
+    // NOTE: This default is also applied in a-scene.js.
+    var attrString = HTMLElement.prototype.getAttribute.call(el, this.name);
+    if (attrString && !attrString.match(/($|\W)antialias\W/)) {
+      el.setAttribute(this.name, {antialias: !el.isMobile});
```

Comment:
Can do `setAttribute('renderer', 'antialias', `!el.isMobile`);`

Don't need `this.name` above either, can just be `renderer`

Reply:
Done